### PR TITLE
Explain db_data.sql

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Explain the behavior of file db_data.sql . Also rewrite a few items.
| Fixed ticket? |

### CODE REVIEW

File: src/content/1.7/development/database/structure.md [hunk 1]
```diff
@@ -44,7 +46,7 @@ file will be stored in the folder `/install/upgrade/sql/`.
 
 Its name is the PS version on which the change will be applied.
 
-For instance, here is the file *[1.7.3.0.sql](https://github.com/PrestaShop/PrestaShop/blob/1.7.3.x/install-dev/upgrade/sql/1.7.3.0.sql)*,
+For instance, here is the file *[1.7.3.0.sql](https://github.com/PrestaShop/PrestaShop/blob/1.7.3.0/install-dev/upgrade/sql/1.7.3.0.sql)*,
```
Comment (matks) [inline src/content/1.7/development/database/structure.md:49]: This link is more likely to always be valid

File: src/content/1.7/development/database/structure.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ modified, this the file you should open.
 
 ### Table in DB but not in db_structure.sql
 
-With PrestaShop 1.7, some tables have been migrated to doctrine entities
+With PrestaShop 1.7, some tables are now [Doctrine](https://www.doctrine-project.org/) entities
```
Comment (Progi1984) [inline src/content/1.7/development/database/structure.md:20]:
```suggestion
With PrestaShop 1.7, some tables are now linked to [Doctrine](https://www.doctrine-project.org/) entities
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback

**Review by Progi1984 — APPROVED** [review]

**Comment by matks** [comment]
Thank you for review @Progi1984

**Pull request merged**